app/src/main/java/org/mozilla/fenix/components/toolbar/FenixTabCounterToolbarButton.kt

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.components.toolbar
+
+import android.content.Context
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.content.ContextCompat
+import androidx.lifecycle.LifecycleOwner
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import mozilla.components.browser.state.selector.getNormalOrPrivateTabs
+import mozilla.components.browser.state.store.BrowserStore
+import mozilla.components.concept.toolbar.Toolbar
+import mozilla.components.feature.tabs.toolbar.TabCounterToolbarButton
+import mozilla.components.support.ktx.android.content.res.resolveAttribute
+import mozilla.components.support.utils.DrawableUtils
+import mozilla.components.ui.tabcounter.R
+import mozilla.components.ui.tabcounter.TabCounter
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class FenixTabCounterToolbarButton(
+    val lifecycleOwner: LifecycleOwner,
+    filterPrivate: Boolean? = null,
+    val showTabs: () -> Unit,
+    val store: BrowserStore,
+    val tabCounterMenu: FenixTabCounterMenu
+) : Toolbar.Action,
+    TabCounterToolbarButton(
+        lifecycleOwner,
+        filterPrivate,
+        showTabs,
+        store,
+        tabCounterMenu
+    ) {
+
+    override fun createView(parent: ViewGroup): View {
+        val view = super.createView(parent)
+
+        view.apply {
+            addOnAttachStateChangeListener(object : View.OnAttachStateChangeListener {
```

Comment:
This workaround really shouldn't be needed anymore, see https://github.com/mozilla-mobile/android-components/pull/8872#discussion_r516762918

Did you run into problem without this? If so, let's try and fix them.

Reply:
Nope, no problems, just a faulty memory ;)